### DISCUSSION TITLE
TP4 - Pregunta sobre punteros y direcciones

### DISCUSSION THREAD

**Discussion by Elias-Turner** [INGCOM-UNRN-P1/cursada-2023#152] [Preguntas y respuestas] [answered]
Tengo 3 preguntas. Una lleva a la otra.

1) ¿La direccion y el valor son lo mismo?

2) Si la respuesta es NO, entonces ¿El puntero señala al contenido, no?

3) Porque si no es asi, entonces ¿Porque cuando cambias el valor del puntero cambia el valor original? si solo cambia la direccion del puntero, en ese caso no deberia cambiar el valor

**Comment by AgostinaApraiz** [comment, marked as answer]
holaa,
1. No, son distintas. 
2. El puntero señala el lugar de memoria donde este el contenido que vos estes apuntando. 
3. Cambia por que estas cambiando de casillero de memoria.
